mappings/net/minecraft/world/gen/feature/ConfiguredFeatures.mapping

```diff
@@ -7,6 +7,8 @@ CLASS net/minecraft/class_5464 net/minecraft/world/gen/feature/ConfiguredFeature
 	FIELD field_29088 LAPIS_ORE_TARGETS Lcom/google/common/collect/ImmutableList;
 	FIELD field_29090 IRON_CONFIG Lnet/minecraft/class_3124;
 	FIELD field_29091 REDSTONE_CONFIG Lnet/minecraft/class_3124;
+	FIELD field_29511 RANDOM_BERRIES_CAVE_VINES_BODY_PROVIDER Lnet/minecraft/class_4657;
+	FIELD field_29512 RANDOM_AGE_CAVE_VINES_HEAD_PROVIDER Lnet/minecraft/class_5929;
```

Comment:
Can the names be more concise?

Reply:
Should the `RANDOM_X_` prefix be removed? I don't think any other parts of the name could be removed without losing information.

Reply:
The random is needed info still. This is fine